### PR TITLE
Update edit document page

### PR DESCRIPTION
**Pull request by alex-ju**
This PR:
- Disables the autocomplete on the edit form
- Updates edit page content
- Replace toggle link with tabs for markdown preview
  - use `webcomponents-bundle` instead of `webcomponents-loader`, as the loader doesn't contain the polyfills (Firefox and IE fail now because of the lack of this polyfill)
- Replace Markdown toolbar elements with the extended version (this was causing the Heading injectors to fail, because they're not part of the original Github Toolbar)
  - Github doesn't doesn't publish the source code for the markdown toolbar element in its package so we need to publish our own fork to npm. Until then we'll bring the transpiled code in the project.

### Before
ethical standards for public service 1

### After
ethical standards for public service

### CODE REVIEW

File: app/assets/stylesheets/components/_markdown-editor.scss [hunk 1]
```diff
@@ -1,30 +1,50 @@
-.app-c-markdown-editor__preview-toggle {
+.app-c-markdown-editor__head {
   @include govuk-font($size: 19);
 
   display: none;
+  position: relative;
+  z-index: 20;
```
Comment (benthorner) [inline app/assets/stylesheets/components/_markdown-editor.scss]: This causes the focus outline to be hidden and looks a bit weird - is that intentional?

**Comment by kevindew** [comment]
Looks good - I noticed we lose focus indicator on the top of the markdown textarea

screen shot 2018-08-21 at 12 57 45

File: app/assets/stylesheets/components/_markdown-editor.scss (hunk 1 above)
Comment (benthorner) [reply app/assets/stylesheets/components/_markdown-editor.scss]: Also, the outline for the toggle buttons is slightly misaligned with the button borders.

Comment (alex-ju) [reply app/assets/stylesheets/components/_markdown-editor.scss]: It is actually intentional, to position the toolbar over the textarea. I'll try to think of ways of enhancing it.

**Comment by alex-ju** [comment]
@kevindew @benthorner 
Changed the approach now the focus contains the toolbar while preserving tab-index order for accessibility.
screen shot 2018-08-21 at 14 37 45

**Comment by benthorner** [comment]
@alex-ju That looks good, although the button outlines are still weird on my machine.

screen shot 2018-08-21 at 14 47 54

screen shot 2018-08-21 at 14 47 50

**Comment by alex-ju** [comment]
@benthorner just pushed an update to fix that

**Pull request merged**